### PR TITLE
(CL) Add 1h Price Range to CL Historical Chart

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

File: packages/web/components/chart/token-pair-historical.tsx [hunk 1]
```diff
@@ -206,6 +206,11 @@ export const PriceChartHeader: FunctionComponent<{
               classes?.buttons
             )}
           >
+            <ChartButton
+              label="1 hour"
```
Comment (jonator) [inline packages/web/components/chart/token-pair-historical.tsx:210]: Can we localize this?

**Comment by sunnya97** [comment]
Do we still want to do this **right now**, if we're going to use the classical pool chart instead?  We still will need it for new assets later

**Comment by jonator** [comment]
@sunnya97 should we use classical chart or add 1h time range?

**Comment by MaxMillington** [comment]
Any updates on this PR? Should we close it if it isn't actively being worked on?

**Comment by jonator** [comment]
It's working for me: 
Screenshot 2023-08-10 at 5 55 09 PM

@JoseRFelix thoughts?

**Comment by jonator** [comment]
Can reference this branch for adding it in the future, but it is quite stale

**Pull request closed without merging**